fcrepo/4.7.5-demo/bin/entrypoint.sh

```diff
@@ -46,6 +46,18 @@ OPTS="-Dfcrepo.log=${FCREPO_LOG_LEVEL}
       -Dfcrepo.dynamic.stomp.port=${FCREPO_STOMP_PORT}                       \
       -Dcom.arjuna.ats.arjuna.objectstore.objectStoreDir=${ARJUNA_OBJECTSTORE_DIRECTORY}"
 
+if [ -n "${FCREPO_JMX_PORT}" ]
```

Comment:
Is `fcrepo` using or exposing JMX?

Reply:
For development and debugging, I had to go in some really dark places.